readme.md

```diff
@@ -113,6 +114,373 @@ npx actions-up --yes
 npx actions-up -y
 ```
 
+### Dry Run Mode
+
+Check for updates without making any changes:
+
+```bash
+npx actions-up --dry-run
+```
+
+## GitHub Actions Integration
+
+### Automated PR Checks
+
+You can integrate Actions Up into your CI/CD pipeline to automatically check for outdated actions on every pull request. This helps maintain security and ensures your team stays aware of available updates.
+
+Create `.github/workflows/check-actions-updates.yml`:
+
+```yaml
+name: Check for outdated GitHub Actions
+on:
+  pull_request:
+    types: [edited, opened, synchronize, reopened]
+
+jobs:
+  check-actions:
+    name: Check for GHA updates
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install actions-up
+        run: npm install -g actions-up
+
+      - name: Run actions-up check
+        id: actions-check
+        run: |
+          echo "## GitHub Actions Update Check" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+
+          # Initialize variables
+          HAS_UPDATES=false
+          UPDATE_COUNT=0
+
+          # Run actions-up and capture output (force no color output)
+          echo "Running actions-up to check for updates..."
+
+          if [ "${{ inputs.check-mode }}" == "dry-run" ]; then
+            NO_COLOR=1 FORCE_COLOR=0 actions-up --dry-run > actions-up-raw.txt 2>&1 || true
+          else
+            # Run in non-interactive mode to just check
+            NO_COLOR=1 FORCE_COLOR=0 actions-up --dry-run > actions-up-raw.txt 2>&1 || true
+          fi
```

Comment:
It seems that both branches do the same thing, so this if statement is unnecessary?

Reply:
You're right, they are not, fixed it